database/seed.js

```diff
@@ -3,22 +3,45 @@ const { config } = require('../config.js');
 const { createBucket, deleteBucket, uploadFile, listPhotos }  = require('./aws-s3.js');
 const sequelize = require('../database/Models');
 
-const seed = async () => {
+const seed = async (database) => {
 
-  const photos = await listPhotos();
-  const objects = [];
+  const s3Photos = await listPhotos();
+  let rooms = [];
+  let photos = [];
+  let room_number = 0;
 
-  for (let i = 0; i < 1000; i++) {
-    photos.Contents.forEach(photo => {
-      const storage_url = `https://sdc-airbnb-photos.s3.us-east-2.amazonaws.com/${photo.Key}`
-      const name = photo.Key.split('.')[0];
-      const caption = faker.commerce.productName();
-      const is_primary = i % 5 === 0;
-      objects.push({ storage_url, name, caption, is_primary });
-    });
-  };
+  for (let run = 0; run < 10000; run++) {
+    for (let i = 0; i < 5; i++) {
+      s3Photos.Contents.forEach((photo, j) => {
+        if (j % 5 === 0) {
+          const storage_url = `https://sdc-airbnb-photos.s3.us-east-2.amazonaws.com/${photo.Key}`
```

Comment:
I think you could pull out the lines where you define storage_url, name, and caption from your conditional statement as you have that repeating.